Homework6

FrankieChik opened the pull request:
It should be in Homework-06-documentation-testsanswers.ipynb

Commit:
Homework2.ipynb

Commit:
Homework2(This_is_my_answer).ipynb

Commit:
Homework2Finalanswers.ipynb

jdamerow committed:
homework assignment 6

Commit:
03Homeworkanswers.ipynb

FrankieChik committed:
Merge pull request #11 from diging-training/master

homework assignment 6

jdamerow committed:
Added homework for week 7

Commit:
Homeowrk4.ipynb

FrankieChik committed:
Merge pull request #14 from diging-training/master

Added homework for week 7

jdamerow committed:
added homework for week 8

FrankieChik committed:
Create README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

Commit:
Homework-05.ipynb

FrankieChik committed:
Merge pull request #16 from diging-training/master

added homework for week 8

FrankieChik committed:
Create index.md

FrankieChik committed:
Create secondfile

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update secondfile

Commit:
03Homework-answers.ipynb

Commit:
Homework2(This_is_my_answers).ipynb

Commit:
Homework-06-documentation-testsanswers.ipynb

FrankieChik committed:
Create project.py

FrankieChik committed:
Update project.py

jdamerow commented on Homework-06-documentation-testsanswers.ipynb:
```diff
@@ -0,0 +1,201 @@
+    "    \n",
+    "    def test_count_words(self):\n",
+    "        stat=TextStats(text)\n",
+    "        self.assertEqual(stat.count_words(\"butter\"), 2)\n",
```
`count_words` doesn't take any arguments. And if you look at the code, you'll see that it returns a dictionary.

jdamerow commented on Homework-06-documentation-testsanswers.ipynb:
```diff
@@ -0,0 +1,201 @@
+    "\n",
+    "class TextStats:\n",
+    "    \n",
+    "    def __init__(self):\n",
```
It looks like you changed the TextStats class here, why? The next line should throw an error since `text` is not defined.

jdamerow commented on Homework-06-documentation-testsanswers.ipynb:
```diff
@@ -0,0 +1,201 @@
+    "        self.text=text\n",
+    "    \n",
+    "    def test_count_words(self):\n",
+    "        stat=TextStats(text)\n",
```
`text` doesn't really exist at this point, you probably mean `self.text`?

jdamerow commented on Homework-06-documentation-testsanswers.ipynb:
```diff
@@ -0,0 +1,201 @@
+    }
+   ],
+   "source": [
+    "# should be in separate repository.\n",
```
this should all be in separate repository 

> Start working on your project. Create a GitHub repository for your code.

jdamerow commented on Homework-06-documentation-testsanswers.ipynb:
```diff
@@ -0,0 +1,201 @@
+    "        print(name)\n",
+    "with  os.scandir('/Users/hinmingfrankiechik/Desktop/text_files/2009') as dircontent:\n",
+    "    for name in dircontent:\n",
+    "        print(name)\n",
```
all of the above statements are not needed. Your first `os.walk` call gives you all files in all subfolders.

FrankieChik committed:
Create project_code_04082020.py

Hi Julia and Ken,
I went over the program historians and other online topic modeling resources these two days. I am finally able to tokenize and clean the texts I am now working on. I ran the code once on Pycharm but only it came out with some numbers like [(0, 24), (1, 21), (2, 6), (3, 3), (4, 3), (5, 3), (6, 3), (7, 3), (8, 3), (9, 3), (10, 3), (11, 3), (12, 3).......
I think I may need to process topic modeling this week but may I know if I am on the right track? Thanks.
